_layouts/event.html

```diff
@@ -10,27 +10,25 @@
   <header class="post-header">
     <h1 class="post-title p-name" itemprop="name headline">{{ page.title | escape }}</h1>
     <p class="post-meta">
-      <time class="dt-published" datetime="{{ page.date | date_to_xmlschema }}" itemprop="datePublished">
-        {%- assign date_format = site.minima.date_format | default: "%b %-d, %Y" -%}
-        {{ page.date | date: date_format }}
-      </time>•
-      {%- for information in layout.dial_in_information -%}
-        <span itemprop="author" itemscope itemtype="http://schema.org/Person"><span class="p-author h-card" itemprop="name"> {{ information }} •</span></span>
-      {%- endfor -%}
-        <span itemprop="author" itemscope itemtype="http://schema.org/Person"><span class="p-author h-card" itemprop="name"> <a href="{{ layout.dial_in_link }}">Dial In</a></span></span>
+        <span itemprop="author" itemscope itemtype="http://schema.org/Person"><span class="p-author h-card" itemprop="name"> Zoom Dial In Link: <a href="{{ layout.dial_in_link }}">{{ layout.dial_in_link }}</a></span></span>
       </p>
   </header>
 
   <div class="post-content e-content" itemprop="articleBody">
     {{ content }}
-  </div>
-  <h2>Recording</h2>
-    {{ page.recording }}
-    <h2>Presentation</h2>
-    {{ page.presentation }}
+    {%- if page.file_list.size > 0 -%}
```

Comment:
I don't remember what counts as truthy here, but `page.file_list` isn't sufficient?

Reply:
My experience was that it was not sufficient